src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -842,13 +858,22 @@ private static bool IsViableType(LookupResult result)
 
             result.Free();
             return NamespaceOrTypeOrAliasSymbolWithAnnotations.CreateUnannotated(AreNullableAnnotationsEnabled(node.Identifier), bindingResult);
+
+            NamedTypeSymbol getNativeIntType(SyntaxNode node, bool unsigned, DiagnosticBag diagnostics)
+            {
+                if (Compilation.LanguageVersion < MessageID.IDS_FeatureNativeInt.RequiredVersion())
```

Comment:
Should we consider binding these regardless of language version and issuing a language-version error? That way the IDE can offer a refactoring for changing the version.

Reply:
Makes sense.

---
In reply to: [352772214](https://github.com/dotnet/roslyn/pull/39157#discussion_r352772214) [](ancestors = 352772214)